cw_4/src/Pages/Payment/Nav.jsx

```diff
@@ -1,24 +0,0 @@
-import React from 'react'
-import {Radio,RadioGroup,Stack,Text} from "@chakra-ui/react"
-
-const Nav = () => {
-  return (
-    <div className='payment-nav'>
-
-        <div ><img style={{width:"160px"}}  src="https://scontent.fpnq14-1.fna.fbcdn.net/v/t39.30808-6/326521603_1604968816640846_9105269804802444165_n.jpg?_nc_cat=103&ccb=1-7&_nc_sid=730e14&_nc_ohc=ok0kd7s6pq4AX8C5Lu5&_nc_ht=scontent.fpnq14-1.fna&oh=00_AfAgE0evpYYBsmEWxur5_UCLg8YxwIOUIrJDFdYvmH2lHg&oe=63D12D96" alt="logo" /></div>
```

Comment:
this file is looking exactly like CartNav.jsx. can't we reuse CartNav.jsx with necessary tweaks instead of making similar component